plugins/bookmarks/BookmarksSearchEngine.ts

```diff
@@ -1,8 +1,9 @@
-import { IEnvironment, ISearchEngine, SearchResult } from 'einstein'
+import { IEnvironment, ISearchEngine, SearchResult, spawn } from 'einstein'
```

Comment:
suggestion(non-blocking): perhaps we can move `spawn` to a sub namespace under `einstein`? Or we offer everything under `einstein`?

Reply:
we can discuss this later